container-runtime/docker_config.sh

```diff
@@ -23,6 +23,26 @@ set -e ; [ "$TRACE" ] && set -x
 # This script must be run as root.
 [ `id -u` != "0" ] && echo "Must run as root" && exit 1
 
+# read arguments id any
+UNINSTALL=
+RUNTIME_NAME="krun"
+KRUN_PATH="/opt/kontain/bin/krun" 
+
+for arg in "$@"
+do
+   case "$arg" in
+        -u)
+            UNINSTALL=yes
+        ;;
+        --runtime-name=*)
+            RUNTIME_NAME="${1#*=}"
+        ;;
+        --runtime-path=*)
+            KRUN_PATH="${1#*=}"
```

Comment:
Could you verify that KRUN_PATH (the file itself) exists.
It is a pain trying to discover this when docker and its helpers fail because of this.

Reply:
done